cmd/macaroon/brave-talk/premium_prod.yaml

```
@@ -0,0 +1,20 @@
+tokens:
+  - id: "brave-talk-premium-prod sku token v1"
+    version: 1
+    location: "talk.brave.com"
+    first_party_caveats:
+      - sku: "brave-talk-premium-prod"
+      - price: 7.00
+      - currency: "USD"
+      - description: "Premium access to Brave Talk"
+      - credential_type: "single-use"
+      - allowed_payment_methods: "stripe"
+      - metadata: '
+{
+  "stripe_product_id": "prod_Jw4zQxdHkpxSOe",
+  "stripe_item_id": "price_1JICpEBSm1mtrN9nwCKvpYQ4",
+  "stripe_success_uri": "https://account.brave.com/account/?intent=provision",
+  "stripe_cancel_uri": "https://account.brave.com/plans/?intent=checkout"
+}'
```

```suggestion
tokens:
    id: "brave-talk-premium-prod sku token v1"
    version: 1
    location: "talk.brave.com"
    first_party_caveats:
      sku: "brave-talk-premium-prod"
      price: 7.00
      currency: "USD"
      description: "Premium access to Brave Talk"
      credential_type: "single-use"
      allowed_payment_methods: "stripe"
      metadata: >-
                        {
                           "stripe_product_id": "prod_Jw4zQxdHkpxSOe",
                           "stripe_item_id": "price_1JICpEBSm1mtrN9nwCKvpYQ4",
                           "stripe_success_uri": "https://account.brave.com/account/?intent=provision",
                           "stripe_cancel_uri": "https://account.brave.com/plans/?intent=checkout"
                        }
```

(1) I think the dashes were added in the wrong place (they usually instantiate default values in an array.)
(2) I referenced https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines to make a better looking multi-line JSON